src/libc/dlfcn.rs

```diff
@@ -12,18 +12,23 @@ use crate::Environment;
 fn dlopen(env: &mut Environment, path: ConstPtr<u8>, _mode: i32) -> MutVoidPtr {
     // TODO: dlopen() support for real dynamic libraries, and support for all
     // libraries with host implementations.
-    assert_eq!(env.mem.cstr_at_utf8(path), Ok("/usr/lib/libSystem.B.dylib"));
+    assert!([
+        Ok("/usr/lib/libSystem.B.dylib"),
+        Ok("/System/Library/Frameworks/OpenAL.framework/OpenAL")
```

Comment:
Since we are repeating this same list in three places, maybe there should be a `const` for it above? Not important though.

Reply:
Done